src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -405,5 +405,16 @@ public void Dispose()
             State = FunctionInvocationDispatcherState.Disposing;
             Dispose(true);
         }
+
+        public async Task RestartAsync()
```

Comment:
Rename method. RestartFaultedChannelsAsync or RestartErroredChannelsAsync

Reply:
Not applicable anymore.